assets/js/theme/global/object-fit-polyfill.js

```diff
@@ -0,0 +1,8 @@
+import objectFitImages from 'object-fit-images';
+
+export default function () {
+    /* Polyfill object-fit for IE11 only */
+    if (navigator.userAgent.match(/Trident.*rv\:11\./)) {
```

Comment:
Most likely this kind of logic is already in the polyfill code, we shouldn't need it here. In my other comment, I was referring to using some kind of webpack magic to make sure the file is only downloaded when using IE11. This would require moving it out of global and into a separate entry point. However, if this is only 3kb uncompressed, I'm fine with just including it in the main bundle.

Reply:
Okay, I've dropped that change.